### PR TITLE
initial LSF-CSM integration scripts and readme

### PR DESCRIPTION
**Pull request by MichaelBrim**
### Description

Initial support for job-level integration with LSF/CSM using "unifycr" allocation
flag. Created two helper scripts that launch/terminate `unifycrd` on the local node.
These are for use in CSM `privileged_prolog` and `privileged_epilog`, which
execute at job startup and completion, respectively. Also created a `README.md`
describing the necessary integration steps.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### CODE REVIEW

**Comment by MichaelBrim** [comment]
WIP NOTE: I'm working on getting access to a Summit TDS where I can make sure the scripts are working properly with LSF/CSM.

File: util/scripts/lsfcsm/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+(i.e., `unifycrd`) on a node. The scripts are:
+* launch: `<INSTALL_PREFIX>/sbin/unifycr_lsfcsm_prolog`
+* terminate: `<INSTALL_PREFIX>/sbin/unifycr_lsfcsm_epilog`
+
+Integration of the helper scripts requires editing the CSM prologue and
+epilogue scripts to support a new "unifycr" job allocation flag.
+Users may specify the allocation flag as an option to the `bsub`
+command:
+```shell
+[prompt]$ bsub -alloc_flags unifycr <other options> jobscript.lsf
```
Comment (boehms) [inline util/scripts/lsfcsm/README.md]:
```suggestion
[prompt]$ bsub -alloc_flags "unifycr <other options>" jobscript.lsf
```

If I remember correctly multiple alloc flags need to be specified in quotes.

Comment (MichaelBrim) [reply util/scripts/lsfcsm/README.md]: The `<other options>` are other bsub options, not additional allocation flags. I'll make that more clear.

File: util/scripts/lsfcsm/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```
+Alternatively, the job script may contain the flag as a `#BSUB`
+directive:
+```shell
+#! /bin/bash -l
+#BSUB -alloc_flags UNIFYCR
```
Comment (boehms) [inline util/scripts/lsfcsm/README.md]:
```suggestion
#BSUB -alloc_flags "UNIFYCR <other options>"
```

File: util/scripts/lsfcsm/README.md (hunk 1 above)
Comment (fwang2) [reply util/scripts/lsfcsm/README.md]: I have jumped through all the hoops and get the final nod from Tundra gatekeeper. I expect to have full access to the machine and getting this started later this week, hopefully.

**Comment by MichaelBrim** [comment]
Feiyi and I finally tested this on Tundra (Summit TDS), and I've made a couple small improvements.

**Comment by dsikich** [comment]
@MichaelBrim so at Oak Ridge did you need a sys admin to install these scripts for you? This is really nice to launch the daemon when you run bsub but it is too bad we need to have root to integrate with the resource manager this way.

**Comment by MichaelBrim** [comment]
@dsikich yes we needed root to install the LSF-CSM scripts and to add the exes/libs to the compute node image.

**Comment by adammoody** [comment]
Thanks, @MichaelBrim and @boehms .

**Pull request merged**